Add support for build args for Driver Docker Image (#374)
neo4j-drivers/testkit: README.md

    Username used to connect, defaults to neo4j
  * `TEST_NEO4J_PASS`  
    Password used to connect

## Sending build args for the Driver Docker Image

> Testkit is able to send diffent `--build-arg` for building the driver docker image.

Reviewer: ```suggestion
TestKit is able to send different `--build-arg`s for building the driver docker image.
```

> This configuration is done by setting environment variables prefixed by

Reviewer: ```suggestion
This configuration is done by setting environment variables prefixed with
```

> `TESTKIT_DRIVER_BUILD_ARG_`, driver build will called with all build args defined
> removing the prefix from the name.

Reviewer: ```suggestion
`TESTKIT_DRIVER_BUILD_ARG_`. The driver build will called with all build args
minus the prefix.
```

For example:

```console
export TESTKIT_DRIVER_BUILD_ARG_NODE_VERSION=12
export TESTKIT_DRIVER_BUILD_ARG_NPM_VERSION=7
python main.py
```

> It should result in a build command line like this:

Reviewer: ```suggestion
will result in the following build command:
```

```console

> ['docker', 'build', '--build-arg', 'NODE_VERSION=12','--build-arg', 'NPM_VERSION=7', '--tag', 'drivers-javascript:local', '/driver/path/testkit']

Reviewer: ```suggestion
docker build --build-arg NODE_VERSION=12 --build-arg NPM_VERSION=7 --tag <some_tag> /driver/path/testkit
```
